### PR TITLE
fix(ADRs): Correct WebComponents section

### PR DESCRIPTION
**Pull request by ExE-Boss**
Firefox is [migrating from XBL to WebComponents](https://mozilla.github.io/firefox-browser-architecture/text/0007-xbl-design-review-packet.html).

See&nbsp;also [bug&nbsp;1397874](https://bugzilla.mozilla.org/show_bug.cgi?id=1397874) and&nbsp;https://github.com/mdn/sprints/issues/967.

### CODE REVIEW

File: ADRs/001-use-react.md [hunk 1]
```diff
@@ -119,14 +119,19 @@ way that React and Angular do. I did not investigate this framework
...
-Chrome, and so we would have to adopt the Polymer framework.
+Web components are the standard component system for the web,
+and in that sense they seem like the best choice for Mozilla.
 
-If we didn't have such ambitous goals for 2019 and weren't expecting
+Firefox is [migrating from XBL to WebComponents](https://mozilla.github.io/firefox-browser-architecture/text/0007-xbl-design-review-packet.html).
+
+##### Polymer
+
+Polymer versions before 3.x aren't well supported by browsers
```
Comment (justinfagnani) [inline ADRs/001-use-react.md]: This is not entirely correct. Polymer 2.x uses Custom Elements v1 and Shadow DOM v1 as supported by Safari, Chrome and Firefox.

**Comment by davidflanagan** [comment]
ADRs are "decision records". We use them to record why we made a decision at the time we made it. If a decision was based on inaccurate information that is still the context in which it was made, and should be recorded as such.

If this ADR was a google doc, I think we could just add notes in the margin correcting the record. But I don't see a good way to do that with github-based ADRs (I'm just figuring this ADR workflow out). So I think that the only way to fix this is to create a new ADR that supersedes this one, and then mark this one as outdated.

We did just amend this ADR yesterday to add some hyperlinks to existing text. But that was strictly a clarification and did not change the meaning in any way.

So I'm going to close this PR and will instead open a new one for a superseding ADR.

**Comment by davidflanagan** [comment]
Actually, I'm not able to close it. @jwhitlock could you close this one, please?

**Comment by ExE-Boss** [comment]
That was some really in&nbsp;accuare information that you based this decision&nbsp;on, _bravo&nbsp;&lt;/sarcasm&gt;_.

See&nbsp;also https://github.com/mdn/sprints/issues/967

**Comment by davidflanagan** [comment]
@ExE-Boss I don't actually think that the information was terribly inaccurate. And your sarcasm is inappropriate and unhelpful here.

**Comment by ExE-Boss** [comment]
Yeah, well [**ADR&#x2011;002**](https://github.com/mdn/mdn/blob/master/ADRs/002-use-flow-for-typechecking.md) isn’t very accurate either, especially this part: 
> - [Flow is more like a linter that checks types and strips type annotations leaving standard JavaScript. TypeScript is more like a compiler for a new language. If we were to choose TypeScript we'd also have to decide which of its non-standard language extensions we will use and which we will not allow.](https://github.com/mdn/mdn/blob/d621e594aaa5a22d8700479eedb54db44b766af4/ADRs/002-use-flow-for-typechecking.md#L132-L134)

Because Flow also uses non&#x2011;standard syntax, and unlike TypeScript, squats on the `.js` extension and `application/javascript` mime&#x2011;type, which are big NO&#x2011;NOs for&nbsp;inter&#x2011;operability.

**Pull request closed without merging**